### DISCUSSION TITLE
Add support to Python and other languages

### DISCUSSION THREAD

**Discussion by GrimPixel** [bcakmakoglu/vue-flow#762] [Ideas]
Languages like Python has scientific libraries. I guess they may be useful here.

**Comment by bcakmakoglu** [comment]
I'm not sure what this post is supposed to tell me? What libraries? What would be useful here? 
Could you elaborate a little more?

**Reply by sefgit** [reply]
Please have a look here [Nodezator](https://github.com/IndiePython/nodezator)

Very powerful, hopefully its node graph could be improved by using vue-flow to improve usability, which I guess won't be easy due to the dynamic nature of Nodezator's nodes, i.e. nodes are automatically defined based on python's functions (defs) declaration.

PS: It's not really related to vue-flow, since IMHO, vue-flow should stay as generic UI / frontend framework to whatever solution needing graphs/nodes interaction.

**Comment by GrimPixel** [comment]
I mean, if a node can be a Python function, it would be more helpful, because Python libraries are more complete for scientific computations.

**Reply by bcakmakoglu** [reply]
Mh... I still don't follow.
You can already implement custom nodes that handle whatever you want them to, that's really just up to you.
What would you expect this library to do for you in this case? A new node type that somehow handles Python?
Seems a lot like a case for custom nodes, dependant on your specific implementation =)

**Reply by GrimPixel** [reply]
I have a Python file, where there is a function. I would like to import such functions as nodes.
e.g.
```
def area___base__height(b, h):
	return b * h / 2
```
I would like to register it as node “area___base__height”.

**Reply by bcakmakoglu** [reply]
I don't think this is a use-case that needs coverage from the library or can even be provided in a proper generic way that makes sense.
Best way to proceed here is to just use [custom nodes](https://vueflow.dev/examples/nodes/) and implement whatever functionality you need from there.

**Reply by GrimPixel** [reply]
Thank you for your help. I will try to figure out how to run Python code through JavaScript.

**Comment by s71m** [comment]
@GrimPixel 
you can take a look - https://github.com/zauberzeug/nicegui/discussions/4365
